frontend/common/src/helpers/formUtils.ts

```diff
@@ -57,3 +57,37 @@ export function enumToOptions<T>(
 export function getValues<T>(list: { value: T; label: string }[]): T[] {
   return list.map((x) => x.value);
 }
+
+/**
+ * Returns a boolean indicating if the compare string matches the word being searched for (needle).
+ * @param needle String that you want to search for.
+ * @param compareString A single string to check.
```

Comment:
I would expect a `needle` var to have a `haystack` (the half-metaphor seems a little confusing otherwise)

Reply:
Yeah, I scratched my head on that one for a while. 😁 For each iteration through the haystack you're comparing the needle to ... ~pieceOfJay~?  ~hayStalk~?  ~oneHay~?